akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst

```diff
@@ -386,6 +386,26 @@ have explicitly relied on this behaviour, however if you find yourself with an a
 should rewrite it to explicitly store the ``ActorPath`` of where such replies during replay may have to be sent to,
 instead of relying on the sender reference during replay.
 
+PersistentView is deprecated
+----------------------------
+
+``PersistentView`` is deprecated. Use :ref:`persistence-query-scala` instead. The corresponding
+query type is ``EventsByPersistenceId``. There are several alternatives for connecting the ``Source``
+to an actor corresponding to a previous ``PersistentView`` actor:
+
+* `Sink.actorRef`_ is simple, but has the disadvantage that there is no back-pressure signal from the 
+  destination actor, i.e. if the actor is not consuming the messages fast enough the mailbox of the actor will grow
+* `mapAsync`_ combined with :ref:`actors-ask-lambda` is almost as simple with the advantage of back-pressure
+  being propagated all the way
+* `ActorSubscriber`_ in case you need more fine grained control
+  
+The consuming actor may be a plain ``Actor`` or a ``PersistentActor`` if it needs to store its
+own state (e.g. fromSequenceNr offset).
+
+.. _Sink.actorRef: http://doc.akka.io/docs/akka-stream-and-http-experimental/1.0/scala/stream-integrations.html#Sink_actorRef
+.. _mapAsync: http://doc.akka.io/docs/akka-stream-and-http-experimental/1.0/stages-overview.html#Asynchronous_processing_stages
+.. _ActorSubscriber: http://doc.akka.io/docs/akka-stream-and-http-experimental/1.0/scala/stream-integrations.html#ActorSubscriber
+
```

Comment:
Very good summary of options, (and :+1: on including the entire text in the docs above)